torch/_dynamo/variables/torch.py

```diff
@@ -481,9 +481,34 @@ def get_state_from_generator():
             if self.value == torch._C._nn.scaled_dot_product_attention:
                 # See:[Note] SDPA_flash's meta function returns incorrect Philox seed and offset
                 # in pytorch/torch/_meta_registrations.py
-                fake_query = args[0].as_proxy().node.meta["example_value"]
-                fake_key = args[1].as_proxy().node.meta["example_value"]
-                fake_value = args[2].as_proxy().node.meta["example_value"]
+                all_kwargs = kwargs.copy()
+                all_kwargs.update(
+                    dict(
+                        zip(
+                            (
+                                "query",
+                                "key",
+                                "value",
+                                "attn_mask",
+                                "dropout_p",
+                                "is_causal",
+                            ),
+                            args,
```

Comment:
add a quick test - so we catch issues if e.g. someone adds an arg in the native_function schema and forgets to update this code?

Reply:
totally agree, `pytorch/test/dynamo/test_misc.py` does this seem like a reasonable place to put the test?